src/oci.h

```diff
@@ -137,6 +137,11 @@
 /* Path to the stateless passwd file. */ 
 #define STATELESS_PASSWD_PATH "/usr/share/defaults/etc/passwd"
 
+/* Offset to add to the interface index for assigning the pci slot.
+ * First 3 slots are in use for pc-lite machine type
+ */
+#define PCI_OFFSET 4
```

Comment:
We should correlate if this is at all affected/related to the PCI BAR scanning optimisations in the guest kernel as well - if you look at the guest kernel patches you'll find:

0006-probe-patch
407-pci-guest....

where we optimise the BAR scanning for the pc-lite machine, I believe to 4 slots presently, as we 'know' there is nothing beyond there ... or wasn't.

Indeed, it would be good to get @anthonyzxu input here.
(some of those two should really be merged ;-)